packages/foam-vscode/src/features/document-decorator.ts

```diff
@@ -14,34 +14,59 @@ const placeholderDecoration = vscode.window.createTextEditorDecorationType({
   cursor: 'pointer',
 });
 
+const tagDecoration = vscode.window.createTextEditorDecorationType({
+  rangeBehavior: vscode.DecorationRangeBehavior.ClosedClosed,
+  textDecoration: 'none',
+  color: { id: 'foam.tag' },
+  cursor: 'pointer',
```

Comment:
how come a pointer cursor?

Reply:
It wasn't really intentional, just an artifact from the links object. I've removed.